ai/core/tests/core/databricks/test_databricks_integration_tests.py

```diff
@@ -188,30 +153,6 @@ def test_execute_function_using_serverless_row_limit(
         assert result.truncated is True
 
 
-@retry_flaky_test()
-@requires_databricks
-def test_execute_function_with_timeout(client: DatabricksFunctionClient, monkeypatch):
```

Comment:
This test only validates warehouse execution env variable overrides. It is not needed any longer